### DISCUSSION TITLE
DHCP设置自定义IPv4 DNS服务器无效

### DISCUSSION THREAD

**Discussion by rawvoid** [immortalwrt/immortalwrt#1048] [Q&A]
### Describe the bug

Hi,

我将`Dnsmasq`的DNS端口设置成`530`，使用`ADGuardHome`作为默认的DNS服务器，这样设置之后发现局域网内其他设备获取不到IPv4 DNS服务器地址，IPv6正常。在网上查相关资料之后，发现一个解决方案是在`DHCP 选项`中设置：`6,局域网网关`，如下图所示：

![image](https://github.com/immortalwrt/immortalwrt/assets/70388114/36c90c01-d739-44d1-8e53-8293e6a14ab0)

但是问题依旧无法获取IPv4 DNS地址

![image](https://github.com/immortalwrt/immortalwrt/assets/70388114/ddbc4283-c2d0-4234-82c2-90b0b5ac1cd4)

### ImmortalWrt version

r26836-a59b9d368e

### ImmortalWrt target/subtarget

x86/64

### Device

O.E.M O.E.M

### Image kind

Official downloaded image

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
如果你真的这么喜欢 adguardhome，让 adguardhome 一起接管 DHCP 服务即可。

**Comment by rawvoid** [comment]
Hi,

> 如果你真的这么喜欢 adguardhome，让 adguardhome 一起接管 DHCP 服务即可。

是我的问题，重启电脑就好了。谢谢
